packages/site/src/components/ContentLoader.tsx

```diff
@@ -28,11 +28,5 @@ export const ContentLoader = () => {
 
   const content = contentMap[type][name];
 
-  return (
-    <ContentView
-      intro={content.intro}
```

Comment:
intro and title were not being used. We should consider removing these from the mappings unless we plan to use them later?

Reply:
Interesting - I didn't realize we don't need them in the maps. But good call on using title for the tabs

Reply:
It seems like something we thought we would need but at the moment it doesn't seem like it we are. Something we could get rid of at the end when we are doing other cleanup